blueprints/banking_blueprint/domains/accounts_creditcards/credit_due_date/train.txt

```diff
@@ -0,0 +1,122 @@
+How long do I have to pay my credit card bill
```

Comment:
Label the entity 'credit card'? Throughout this file and the others below? Otherwise the recognizer won't be able to detect it at inference time for these intents. You can skip them for FAQ intents if entities aren't needed there.